### PR TITLE
https://issues.redhat.com/browse/ACM-8727

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-8727

### CODE REVIEW

File: release_notes/known_issues_application.adoc [hunk 1]
```diff
@@ -29,6 +29,24 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
...
+
+[source,yaml]
+----
+credentials:
+  secretName: dbsecret
+  usernameSecretKey: username
+  passwordSecretKey: password
+----
+
+The helm chart with secret dependencies is only supported in the helm binary CLI. It is not supported in the operator SDK helm library. The {product-title-short} subscription controller applies the operator SDK helm library to install and upgrade the helm chart. Therefore, the {product-title-short} subscription cannot deploy the helm chart with secret dependencies.
```
Comment (swopebe) [inline release_notes/known_issues_application.adoc]: see third-party doc to get the product names documented correctly: https://helm.sh/

File: release_notes/known_issues_application.adoc [hunk 2]
```diff
@@ -29,6 +29,24 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
 
 See the following known issues for the application lifecycle component.
 
+[#helm-chart-with-secret-dependencies-cannot-be-deployed]
+== Helm chart with secret dependencies cannot be deployed by the {product-title-short} subscription 
+//2.9:ACM-8727
+
+Using helm chart, you can define privacy data in a Kubernetes secret and refer to this secret within the `value.yaml` of the helm chart.  
+
+The username and password are given by the referred Kubernetes secret resource `dbsecret`. For example, see the following sample `value.yaml`:
```
Comment (swopebe) [inline release_notes/known_issues_application.adoc]: This needs to be completed:

See the `value.yaml` file.

File: release_notes/known_issues_application.adoc [hunk 3]
```diff
@@ -29,6 +29,24 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
 
 See the following known issues for the application lifecycle component.
 
+[#helm-chart-with-secret-dependencies-cannot-be-deployed]
+== Helm chart with secret dependencies cannot be deployed by the {product-title-short} subscription 
+//2.9:ACM-8727
+
+Using helm chart, you can define privacy data in a Kubernetes secret and refer to this secret within the `value.yaml` of the helm chart.
```
Comment (swopebe) [inline release_notes/known_issues_application.adoc]: `value.yaml` file. I think this is mentioned in IBM manual, as well.

File: release_notes/known_issues_application.adoc [hunk 4]
```diff
@@ -29,6 +29,24 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
 
 See the following known issues for the application lifecycle component.
 
+[#helm-chart-with-secret-dependencies-cannot-be-deployed]
+== Helm chart with secret dependencies cannot be deployed by the {product-title-short} subscription 
+//2.9:ACM-8727
```
Comment (swopebe) [inline release_notes/known_issues_application.adoc:34]: Is it just the "RHACM" subscription @xiangjingli ?  (Or does it have another name?)

File: release_notes/known_issues_application.adoc (hunk 3 above)
Comment (swopebe) [reply release_notes/known_issues_application.adoc]: https://www.ibm.com/docs/en/ibm-style?topic=elements-files-directories

**Review by swopebe — CHANGES_REQUESTED** [review]

File: release_notes/known_issues_application.adoc (hunk 1 above)
Comment (xiangjingli) [reply release_notes/known_issues_application.adoc]: The `operator SDK helm library`  seems appropriate term 

It is from the operator SDK project https://github.com/operator-framework/operator-sdk
and the helm library we leveraged is from its internal helm lib

https://github.com/operator-framework/operator-sdk/tree/master/internal/helm

File: release_notes/known_issues_application.adoc (hunk 4 above)
Comment (xiangjingli) [reply release_notes/known_issues_application.adoc:34]: Yes, it is just the issue in the RHACM subscription

File: release_notes/known_issues_application.adoc (hunk 3 above)
Comment (swopebe) [reply release_notes/known_issues_application.adoc]: This should be as shown in the style guide:

... within the `value.yaml` _file._

Comment (swopebe) [reply release_notes/known_issues_application.adoc]: File only italicized for emphasis here, not in the actual doc.

File: release_notes/known_issues_application.adoc (hunk 2 above)
Comment (swopebe) [reply release_notes/known_issues_application.adoc]: same here.

**Review by swopebe — APPROVED** [review]
Looks great, it took a while for me to see your changes but I finally did. Made one small change to the operator name bc one instance was lower case, the other was not. Either way, we should be consistent.

**Pull request merged**